### PR TITLE
Add support for CN

### PR DESCRIPTION
**Pull request by gamous**

### CODE REVIEW

File: FishNotify.csproj [hunk 1]
```diff
@@ -11,14 +11,14 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net5.0-windows</TargetFramework>
```
Comment (carvelli) [inline FishNotify.csproj:14]: This breaks compatibility with the non-CN dalamud version, and as such this can't be merged.

**Comment by carvelli** [comment]
Sorry for the late response, but the fact it breaks the global version of the game means I won't be able to merge this :(

**Comment by carvelli** [comment]
Hey,

Hope you'll see this notification - there is a beta version with CN support now available at https://github.com/carvelli/Fish-Notify/releases/download/v5.0/FishNotify-5.0-CN.zip

Please test it and let me know if it works.

**Comment by gamous** [comment]
I'm sorry it doesn't work in CN Server.
```
18:00:59.470 | ERR [LOCALPLUGIN] Error while loading Fish Notify
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at FishNotify.FishNotifyPlugin..ctor() in E:\ffxiv\FishNotify\FishNotify.cs:line 40
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeConstructorInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Dalamud.IoC.Internal.ServiceContainer.CreateAsync(Type objectType, Object[] scopedObjects)
   at Dalamud.Plugin.Internal.Types.LocalPlugin.LoadAsync(PluginLoadReason reason, Boolean reloading)
--------------------------------
18:00:59.479 | ERR [PLUGINW] Plugin installer threw an unexpected error
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at FishNotify.FishNotifyPlugin..ctor() in E:\ffxiv\FishNotify\FishNotify.cs:line 40
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeConstructorInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Dalamud.IoC.Internal.ServiceContainer.CreateAsync(Type objectType, Object[] scopedObjects)
   at Dalamud.Plugin.Internal.Types.LocalPlugin.LoadAsync(PluginLoadReason reason, Boolean reloading)
{ } [ Send ]
```

**Pull request closed without merging**